docs/kubefirst/gitlab/gitlab.md

```diff
@@ -24,11 +24,11 @@ If you ever run into an issue on a GitLab upgrade or gitops operation that chang
 
 However since GitLab **is the git source of truth** for your GitOps, you cannot roll back GitLab by reverting the commit because git itself may be inaccessible to ArgoCD. Should you fall into this condition, to conduct a rollback to a prior state, go to the `gitlab` application in Argo CD and click the `History and Rollback` button in the header.
```

Comment:
```suggestion
However since GitLab **is the Git source of truth** for your GitOps, you cannot roll back GitLab by reverting the commit because Git itself may be inaccessible to Argo CD. To remedy this, conduct a rollback to a prior state, go to the `gitlab` application in Argo CD and click the `History and Rollback` button in the header.
```